Activities/Prac 5/hex_colours.py

```diff
@@ -0,0 +1,29 @@
+"""
+CP1404 Practical 5
+Hex Colour Dictionary
+"""
+
+HEX_COLOUR_CODES = {"cornflowerblue": "#6495ed", "forestgreen": "#228b22", "yellow1": "#ffff00",
+                    "darkorange": "#ff8c00", "firebrick2": "#ee2c2c", "mediumOrchid3": "#7a378b", "gray9": "#171717",
+                    "floralwhite": "#fffaf0", "deeppink3": "#8b0a50", "saddlebrown": "#8b4513"}
+
+print(HEX_COLOUR_CODES)
+
+
+def main():
+    longest_colour_name = 0
+    for key in HEX_COLOUR_CODES:
+        if len(key) > longest_colour_name:
+            longest_colour_name = len(key)
```

Comment:
Do you think this is relevant? I don't know, I'm sitting on the fence...

Reply:
fancy formatting for the print statement. I don't know about you but this task seemed too much like a copy of the previous.